gyp info it worked if it ends with ok
gyp info using node-gyp@9.0.0
gyp info using node@16.15.1 | linux | x64
gyp info find Python using Python version 3.10.4 found at "/usr/bin/python3"
gyp info spawn /usr/bin/python3
gyp info spawn args [
gyp info spawn args   '/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/home/dev/Desktop/SelfLearning/nftgraphtwo/marketplace-two/node_modules/ursa-optional/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/dev/.cache/node-gyp/16.15.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/home/dev/.cache/node-gyp/16.15.1',
gyp info spawn args   '-Dnode_gyp_dir=/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/home/dev/.cache/node-gyp/16.15.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/home/dev/Desktop/SelfLearning/nftgraphtwo/marketplace-two/node_modules/ursa-optional',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp ERR! build error 
gyp ERR! stack Error: not found: make
gyp ERR! stack     at getNotFoundError (/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:10:17)
gyp ERR! stack     at /home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:57:18
gyp ERR! stack     at new Promise (<anonymous>)
gyp ERR! stack     at step (/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:54:21)
gyp ERR! stack     at /home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:71:22
gyp ERR! stack     at new Promise (<anonymous>)
gyp ERR! stack     at subStep (/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:69:33)
gyp ERR! stack     at /home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/which/which.js:80:22
gyp ERR! stack     at /home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/isexe/mode.js:8:5
gyp ERR! System Linux 5.15.0-46-generic
gyp ERR! command "/home/dev/.nvm/versions/node/v16.15.1/bin/node" "/home/dev/.nvm/versions/node/v16.15.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/dev/Desktop/SelfLearning/nftgraphtwo/marketplace-two/node_modules/ursa-optional
gyp ERR! node -v v16.15.1
gyp ERR! node-gyp -v v9.0.0
gyp ERR! not ok 
